src/common/syncjournaldb.cpp

```diff
@@ -970,6 +976,43 @@ bool SyncJournalDb::setFileRecord(const SyncJournalFileRecord &_record)
     }
 }
 
+void SyncJournalDb::keyValueStoreSet(const QString &key, qint64 defaultValue)
```

Comment:
@FlexW I think the name `defaultValue` is a bit misleading in the case of a setter. There is no default value to this parameter and you're not expecting to use it in case the value is not found in the database as you do with a getter. So, maybe just `value` is enough?

Reply:
Yes, of course. Copy and past error.